TT_SStt_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+#import model Zp_2LLP
+#generate p p > zp > s2 s2~ / x2 s2 j
+import model eVLQS012_v8_4FNS_UFO
+define sx = a z w+ w-
+#define sx = a z
+define tpx = tp tp~
+define tx = t t~
+define bx = b b~
+define lx = e- mu- e+ mu+ ve vm ve~ vm~
+define qx = u c d s u~ c~ d~ s~ 
+define phix = s0 z h w+ w-
+define tbx = t t~ b b~
+define wx = w+ w-
+generate p p > tp tp~, ( tp > s0 t, ( s0 > a a ) , ( t > qx qx bx ) ), ( tp~ > s0 t~, ( t~ > qx qx bx ) )
+#DP FOR GENERATING T' CROSS SECTIONS
+#generate p p > tp tp~
+output TT_SStt -nojpeg
```

Comment:
please remove the commented lines. By using the decay chain syntax, what's the size of the gridpack and how many files after untaring the gridpack

Reply:
Size of gridpack: 22917232, number of files after untaring the gridpack: 2074

Reply:
so the size is around 23M?

Reply:
Hi! To answer your question re size - Yes. I do have a question for you! In the customizecards.dat I included the masses which are to be varied across the 2d grid, but I did not include the widths and coupling values corresponding to our param_card.dat.   Should I have included these?  Please let me know if I need to update the customizecards.dat!  Thank you for all your help!

Reply:
Hello experts!  I would like to add my param_card.dat, but I am not sure where I should put the card?  I no longer see any of my cards?!?  I wonder if you might be able to give me a status for TT_SStt cards?  Thanks so much!